vcsutils/webhookparser/webhookparser.go

```diff
@@ -45,6 +45,34 @@ type WebhookInfo struct {
 	Author WebHookInfoUser `json:"author,omitempty"`
 	// CompareUrl is HTML URL to see git comparison between commits (Push event only)
 	CompareUrl string `json:"compare_url,omitempty"`
+	// PullRequest encapsulates information of the pull request.
+	PullRequest *PullRequestInfo `json:"pull_request,omitempty"`
+}
+
+// PullRequestInfo contains information about pull request.
+type PullRequestInfo struct {
+	// ID is a unique identifier of the pull request.
+	ID int `json:"id"`
+	// Title is a title(name) of the pull request.
+	Title string `json:"title,omitempty"`
+	// Href is a hyperlink to the pull request.
+	Href string `json:"href"`
```

Comment:
Didn't we use `URL` for this data in the "commit push" event?

Reply:
Renamed to Url as in other types.